### PR TITLE
📝 update README

### PR DESCRIPTION
**Pull request by eliascodes**
* Remove old information & images from README
* Add documentation on project stack

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,29 +18,25 @@ On subsequent visits to the community business, the user can quickly sign in wit
...
+* [@eliascodes](https://github.com/eliascodes)
 
 ## Thanks
-
 * Our Mothers and Fathers for bringing us into this world, and supporting us through our mad *I'm a developer now!* epoch.
-* Tim Berners Lee for inventing the internet; thanks  for our addiction dude.
+* Tim Berners Lee for inventing the internet; thanks for our addiction dude.
 * SimplyFresh for sustaining us through the high and the low, the hummus and the chocolate.
 * Delonghi our trusty coffee machine without which we would still be scratching around on the floor like animals.
 * Lastly we would like to thank our talented, tenacious, turbulent and unusually attractive team for all their hard work and commitment. Guys we are the *BEST!*
```
Comment (RogeredBacon) [inline README.md]: I'd be up for removing or at least changing these now this is a professional product. I think this was all written drunkenly on our last day of FAC

Comment (eliascodes) [reply README.md]: Lol, up to you, feel free to make changes to this PR 👍

**Comment by RogeredBacon** [comment]
I'm not adverse to the thanks section being a funny but I think by default it should be professional; so I've removed a couple of lines.

File: README.md [hunk 2]
```diff
@@ -19,29 +18,23 @@ On subsequent visits to the community business, the user can quickly sign in wit
...
+* [@RogeredBacon](https://github.com/RogeredBacon)
+* [@rachaelcodes](https://github.com/rachaelcodes)
+* [@ZooeyMiller](https://github.com/ZooeyMiller)
+* [@finnhodgkin](https://github.com/finnhodgkin)
+* [@astroash](https://github.com/astroash)
+* [@eliascodes](https://github.com/eliascodes)
 
 ## Thanks
-
-* Our Mothers and Fathers for bringing us into this world, and supporting us through our mad *I'm a developer now!* epoch.
```
Comment (astroash) [inline README.md]: It's mothers day in Palestine today. Feels like a kinds cutting time to rip this out of our readme.

**Pull request merged**